src/app/controllers/authentication.server.controller.js

```diff
@@ -100,9 +109,22 @@ exports.verifyPermission = (requiredPermission) =>
    * @param {function} next - Next middleware function
    */
   (req, res, next) => {
-    // Admins always have sufficient permission
-    let hasSufficientPermission =
+    const isFormAdmin =
       String(req.form.admin.id) === String(req.session.user._id)
+
+    // Forbidden if requiredPersmission is admin but user is not
+    if (!isFormAdmin && requiredPermission === PERMISSIONS.ADMIN) {
```

Comment:
could we log the HTTP 403 Forbidden here with the relevant info for ease of troubleshooting in production?

Reply:
i think its better to have another logging middleware

Reply:
to add user, permission and form title in logging

Reply:
TODO: Add a line to log before returning 403 Forbidden

Reply:
added